ddl/cluster.go

```diff
@@ -696,8 +703,9 @@ func finishFlashbackCluster(w *worker, job *model.Job) error {
 	var pdScheduleValue map[string]interface{}
 	var autoAnalyzeValue, readOnlyValue string
 	var gcEnabled bool
+	ttlEnabled := true
 
-	if err := job.DecodeArgs(&flashbackTS, &pdScheduleValue, &gcEnabled, &autoAnalyzeValue, &readOnlyValue, &lockedRegions, &startTS, &commitTS); err != nil {
```

Comment:
Should store session variable in `model.StateNone` state. But one question here, what will happened when ttlEnabled during flashback?

Reply:
done